Rename `WgpuSettingsPriority::Compatibility` to `WgpuSettingsPriority::WebGPU` (#23783)
bevyengine/bevy: _release-content/migration-guides/wgpu_settings_priority_renamed_webgpu.md

---
title: "`WgpuSettingsPriority::Compatibility` renamed to `WgpuSettingsPriority::WebGPU`"
pull_requests: []
---

 `WgpuSettingsPriority::Compatibility` has been renamed to `WgpuSettingsPriority::WebGPU`.

>  And the `WGPU_SETTINGS_PRIO` environment variable needs to be `webgpu` instead of `compatibility`.

Reviewer: ```suggestion
To set this using an environment variable, update `WGPU_SETTINGS_PRIO` to `webgpu` from `compatibility`.
```
